language/stdlib/modules/Libra.move

```diff
@@ -1153,38 +1142,71 @@ module Libra {
     /// aggregate value exceeds the CoinInfo.total_value.  However, that property involves summations over
     /// all resources and is beyond the capabilities of the specification logic or the prover, currently.
 
-    // TODO: This schema is better to be in Roles, but put here to avoid an unused schema warning from Roles.
-    spec schema AbortsIfNotDesignatedDealer {
-        account: signer;
-        aborts_if !Roles::spec_has_designated_dealer_role_addr(Signer::spec_address_of(account));
-    }
-
     spec module {
-        /// The permission "MintCurrency(type)" is granted to TreasuryCompliance [B12].
+        /// The permission "MintCurrency(type)" is granted to TreasuryCompliance [B11].
```

Comment:
We still have to consolidate the notion of "permission" w.r.t. capability, role, etc. The notation `"MintCurrency(type)"` isn't really understandable. Fine if not in this PR but just as a reminder.